sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/src/BlobMetadataKeys.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if PUBLICPROTOCOL
```

Comment:
What is the purpose of the `PUBLICPROTOCOL`variable

Reply:
Not used, I removed it. I also checked webjobs assemblies and there's no trace of "public protocols".
It is somewhat related json serialization/deserialization that we touched upon at our meetings, we'll bring it back if deemed necessary.